changelogs/1.2.0.md

```diff
@@ -31,6 +31,7 @@
   -       return next_directive(parent_node, argument_node, value, ctx)
   +       return next_directive(parent_node, argument_definition_node, argument_node, value, ctx)
   ````
+- A list of non-scalar objects in the output will have it's items resolved sequentially, rather than in parallel.
```

Comment:
Hum, Coercion is sequential but resolution should be done concurrently (if the resolution does an I/O, if not, the gathered coroutines will ran sequentially). @Maximilien-R what your take on this ?

Reply:
On list (input/literal/output coercion) we removed the `asyncio.gather` in order to sequentially `await` loop each item to gain in performance.

Reply:
Yes, but resolution (calling the resolver), it is still gathered ?

Reply:
Hm... As it's implemented right now, I think resolver will be called sequentially too. Since the list coercer will called the `inner_coercer` which will be the child wrapped resolver.

Reply:
Oh yes, I remember it now. Thanks @Maximilien-R for reminding me. @garyd203 Thanks for this PR.